gov-pay-client/src/test/java/uk/gov/hmcts/payment/api/external/client/GovPayClientTest.java

```diff
@@ -30,13 +31,18 @@ public class GovPayClientTest {
 
     @Before
     public void setUp() throws Exception {
+        CloseableHttpClient minimalHttpClient = HttpClients.custom()
+            .disableCookieManagement()
+            .disableAuthCaching()
+            .build();
+
         client = new GovPayClient(
             "http://localhost:" + wireMockRule.port(),
-            HttpClients.createMinimal(),
```

Comment:
HttpClients.createMinimal() removed in HttpClient 5. New minimalHttpClient is set up so it matches old behaviour of HttpClients.createMinimal().